tools/lint-hotkeys

```diff
@@ -26,6 +26,23 @@ HELP_TEXT_STYLE = re.compile(r"^[a-zA-Z /()',&@#:_-]*$")
 KEYS_TO_EXCLUDE = ["q", "e", "m", "r", "Esc"]
 
 
+def read_help_categories() -> Dict[str, List[Tuple[str, List[str]]]]:
```

Comment:
NIT: I feel we can squash this and the previous commit. The previous commit seems redundant since you're refactoring the flow and removing the function anyways.

refactor: lint-hotkeys: Improve variable naming of generated dict.
lint-hotkeys: Refactor the flow by creating new ENTRY_BY_CATEGORIES.

Reply:
Ah, interesting, but we're not actually removing the function, it's just moved to the top to be placed before creating the new ENTRY_BY_CATEGORIES.